lib/themes/dosomething/paraneue_dosomething/js/components/CampaignSignupCard.js

```diff
@@ -30,9 +32,16 @@ class CampaignSignupCard extends React.Component {
         isLoading: false,
         isSignedUp: true
       });
+
+      sendEvent('onboarding-v2', 'signup', this.constructor.name);
     });
   }
 
+  preRefresh() {
```

Comment:
Maybe rename to `preViewMore() { ... }`. Refresh brings a different thing to mind. However, on different note, would it make sense at all to have the event happen back in the controller and instead you just pass params via the `viewMore()` props method to tell it the name of the event `onboarding-v2` and the `this.constructor.name` part... this way the event is always attached to the `viewMore()` method back where it lives?